cypress/e2e/smoke/dbot/checkViewDetails.cy.js

```diff
@@ -0,0 +1,185 @@
+import RunPanel from '../../../support/pageobjects/dbot/run_panel'
+import BotBuilder from '../../../support/pageobjects/dbot/bot_builder_page'
+import BotDashboard from '../../../support/pageobjects/dbot/bot_dashboard_page'
+
+describe('QATEST-4340 - View Details Button on Run Panel', () => {
+  const runPanel = new RunPanel()
+  const botBuilder = new BotBuilder()
+  const botDashboard = new BotDashboard()
+
+  it(`Checking View Details on Transaction tab for mobile'}`, () => {
+    cy.c_login({ user: 'dBot' })
+    cy.c_visitResponsive('/appstore/traders-hub', { size: 'mobile' })
+    cy.c_openDbotThub()
```

Comment:
suggestion: Can skip this and directly go to bot.
```suggestion
    cy.c_visitResponsive('/bot', { size: 'mobile' })
```